### PR TITLE
CY-2393 Use new examples for simple agent upgrade

### PR DESCRIPTION
**Pull request by geokala**

### CODE REVIEW

File: cosmo_tester/framework/examples/__init__.py [hunk 1]
```diff
@@ -20,7 +20,182 @@
...
+
+    def install(self, timeout=900):
+        self.logger.info('Installing deployment...')
+        try:
+            self.cfy.executions.start.install(['-d', self.deployment_id,
+                                               '-t', self.tenant,
+                                               '--timeout', timeout])
+            self.installed = True
+        except Exception as e:
+            if 'if there is a running system-wide' in e.message:
```
Comment (tehasdf) [inline cosmo_tester/framework/examples/__init__.py:141]: plz no e.message

File: cosmo_tester/test_suites/image_based_tests/simple_agent_upgrade_test.py [hunk 2]
```diff
@@ -15,56 +15,65 @@
...
 def test_old_agent_stopped_after_agent_upgrade(
-        managers, nodecellar, cfy, logger, tmpdir
+        managers_and_vm, example, cfy, logger, tmpdir
 ):
     local_snapshot_path = str(tmpdir / 'snapshot.zip')
     snapshot_id = 'snap'
 
-    old_manager = managers[0]
-    new_manager = managers[1]
+    old_manager = managers_and_vm[0]
```
Comment (tehasdf) [inline cosmo_tester/test_suites/image_based_tests/simple_agent_upgrade_test.py:70]: you can btw do `old_manager, new_manager, vm = managers_and_vm` and that of course also implicitly checks that managers_and_vm is indeed 3 entries

File: cosmo_tester/framework/examples/__init__.py (hunk 1 above)
Comment (geokala) [reply cosmo_tester/framework/examples/__init__.py:141]: Ah, sorry- I copied that one from the Abstract one. Will change it.

File: cosmo_tester/test_suites/image_based_tests/simple_agent_upgrade_test.py (hunk 2 above)
Comment (geokala) [reply cosmo_tester/test_suites/image_based_tests/simple_agent_upgrade_test.py:70]: Fixing

**Pull request merged**